src/seaport.ts

```diff
@@ -739,7 +739,10 @@ export class Seaport {
             .slice(2)
             .padStart(64, "0"),
           orderComponents.zoneHash.slice(2),
-          orderComponents.salt.slice(2).padStart(64, "0"),
+          BigNumber.from(orderComponents.salt)
```

Comment:
this will likely save others from issues with having salt represented improperly. Thx!